proof/invariant-abstract/AARCH64/ArchIpc_AI.thy

```diff
@@ -554,11 +585,18 @@ lemma valid_arch_mdb_cap_swap:
             ((caps_of_state s)(a \<mapsto> c', b \<mapsto> c))"
   by (auto simp: valid_arch_mdb_def)
 
+lemma do_ipc_transfer_valid_arch[Ipc_AI_3_assms]:
+  "\<lbrace>valid_arch_state and valid_objs and valid_mdb \<rbrace>
+   do_ipc_transfer s ep bg grt r
+   \<lbrace>\<lambda>rv. valid_arch_state\<rbrace>"
+  by (wpsimp wp: valid_arch_state_lift_aobj_at_no_caps do_ipc_transfer_aobj_at)
+
 end
 
-interpretation Ipc_AI_cont?: Ipc_AI_cont
-proof goal_cases
+interpretation Ipc_AI?: Ipc_AI_3
+  proof goal_cases
```

Comment:
This `proof` indent is wrong. The `proof` keyword goes on the same indent as the lemma/have/interpretation command. I know there are some arguments for indenting it, but there is no point fighting against an actual strong upstream style and mixing things up with yet another style.

Reply:
This one isn't mine, the diff got confused, but I now fixed them on all arches.

Reply:
Thanks, they have been bugging me for a while..